usrp3/tools/scripts/uhd_image_builder_gui.py

```diff
@@ -395,17 +395,12 @@ def file_dialog(self):
         Opens a dialog window to add manually the Out-of-tree module blocks
         """
         append_directory = []
-        filename = QtWidgets.QFileDialog.getOpenFileName(self, 'Open File', '/home/')[0]
```

Comment:
@primercuervo I've replaced the "getOpenFileName" functionality with the "getExistingDirectory" function, which still opens a dialog. The folder selected should be the folder of the OOT files you'd like to include. 

After the folder is opened, it is added to a local list of OOT directories (self.oot_dirs) and the "uhd_image_builder.create_oot_include" function is called with the updated list of OOT directories. This generates the Makefile.OOT.inc file. We then call "self.refresh_oot_dirs" to read the updated Makefile.OOT.inc